src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -18,6 +18,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)AddAccessibilityModifiers\CSharpAddAccessibilityModifiersDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AddBraces\CSharpAddBracesDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AddRequiredParentheses\CSharpAddRequiredPatternParenthesesDiagnosticAnalyzer.cs" />
+    <Compile Include="..\..\..\Analyzers\CSharp\Analyzers\ConvertNamespace\ConvertNamespaceAnalysis.cs" />
```

Comment:
```suggestion
    <Compile Include="$(MSBuildThisFileDirectory)ConvertNamespace\ConvertNamespaceAnalysis.cs" />
```

Reply:
@sharwell  @jasonmalinowski  do you know why manually adding a file added it this way?